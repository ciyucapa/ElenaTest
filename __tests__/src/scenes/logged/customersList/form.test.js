import React from 'react';
import {create} from 'react-test-renderer';

import List from '../../../../../src/scenes/logged/customersList/customerList';

describe('Customer list component', () => {
    test('render Customer list component', () => {
        const tree = create(
            <List
                customers={[
                    {
                        address: 'Address',
                        cellphone: 'Phone',
                        city: 'City',
                        firstName: 'First Name',
                        lastName: 'Last Name'
                    },
                    {
                        address: 'Address 2',
                        cellphone: 'Phone 2',
                        city: 'City 2',
                        firstName: 'First Name 2',
                        lastName: 'Last Name 2'
                    },
                ]}
            />,
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});