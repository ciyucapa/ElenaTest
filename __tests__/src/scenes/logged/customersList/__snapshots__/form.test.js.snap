// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Customer list component render Customer list component 1`] = `
Array [
  <div
    className="sc-eCstlR eYWUDk"
  >
    <div>
      <div>
        Nombre: 
        First Name
      </div>
      <div>
        Apellido: 
        Last Name
      </div>
      <div>
        Direccion: 
        Address
      </div>
      <div>
        Telefono: 
        Phone
      </div>
      <div>
        Ciudad: 
        City
      </div>
    </div>
    <div
      className="sc-jSgvzq fjwiTM"
    >
      <button
        className="sc-iBPTik xaJDg"
        disabled={false}
        onClick={[Function]}
      >
        Editar
      </button>
    </div>
  </div>,
  <div
    className="sc-eCstlR eYWUDk"
  >
    <div>
      <div>
        Nombre: 
        First Name 2
      </div>
      <div>
        Apellido: 
        Last Name 2
      </div>
      <div>
        Direccion: 
        Address 2
      </div>
      <div>
        Telefono: 
        Phone 2
      </div>
      <div>
        Ciudad: 
        City 2
      </div>
    </div>
    <div
      className="sc-jSgvzq fjwiTM"
    >
      <button
        className="sc-iBPTik xaJDg"
        disabled={false}
        onClick={[Function]}
      >
        Editar
      </button>
    </div>
  </div>,
  <button
    className="sc-iBPTik xaJDg"
    disabled={false}
    onClick={[Function]}
  >
    Crear nuevo cliente
  </button>,
]
`;
